### PR TITLE
new event Maker Faire Berlin

### PR DESCRIPTION
**Pull request by github-luckow-org**
about/events

### CODE REVIEW

File: about/events/2019-05-17-maker-faire-berlin.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+layout: Event
+category: event
+title: Maker Faire Berlin
+organizer: Maker Media GmbH
+link: 'https://maker-faire.de/berlin/'
+date: '2018-05-17T23:49:15+02:00'
```
Comment (ghys) [inline about/events/2019-05-17-maker-faire-berlin.md]: Are you living in the past @github-luckow-org? :laughing:

Comment (github-luckow-org) [reply about/events/2019-05-17-maker-faire-berlin.md]: argh. template from last year :)

**Comment by Confectrician** [comment]
Hey @github-luckow-org,
welcome back in the present age. 👅 

Are you using some kind of git client/softwar/tool on your computer and could you fix the dco error if the answer to the first question is yes?

**Comment by github-luckow-org** [comment]
never had a dco error before. for this little task (bringing an event on a webpage) i use github in a browser. any idea how to fix this for a "browser version"?

**Comment by Confectrician** [comment]
>any idea how to fix this for a "browser version"?

No idea, but an answer. -> Not possible as far as i know.
May you throw in your sign-off line in 
`Signed-off-by: your name <your@email.me>`
the first post of this PR.
I will override the error and the the sign-off to the merge commit.

**Comment by github-luckow-org** [comment]
haha :)
i have no idea where to put the Signed-off-by: Stephan Luckow <github@luckow.org> message.

**Comment by kaikreuzer** [comment]
Leave it, adding events is just fine without a sign-off!

**Comment by Confectrician** [comment]
Perfect. Build looks good too.

**Pull request merged**